version-changelog.js

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env node
+
+var readline = require('readline');
+var fs = require('fs');
+var childProcess = require('child_process');
+
+var rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+var pluginName = process.argv[2];
+var pluginSlug = pluginName.substr(3);
+var mainPHPFile = pluginName + '/' + pluginName + '.php';
```

Comment:
I think we should update it always; I tried using `https://codex.wordpress.org/Function_Reference/get_plugin_data` from within `-db.php` to pull the plugin version but apparently that function isn't registered within WordPress when the code executes.  I saw some hacks to manually include files but that seems dangerous because WordPress does move function locations from version to version.